### PR TITLE
EARTH-159 Changed hero banner to top media and added feature image field

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Removed hero banner in favor of top_media field to use hero banner or video
- configured panelizer with new field.
- new featured image field not displayed on node.

# Needed By (Date)
- End of sprint

# Steps to Test

1. Checkout branch and same branch in [stanford_layouts](https://github.com/SU-SWS/stanford_layouts/pull/4) & [SE3_BLT](https://github.com/stanford-earth/SE3_BLT/pull/43)
2. cache clear then Config Import, features import all
3. create a news content
4. populate the hero banner or the video
5. verify it looks correct and the page title displays if video used or hero banner used with nothing in the "super heading" field (with variant set to "page title").

# Associated Issues and/or People
- https://github.com/stanford-earth/SE3_BLT/pull/43
- https://github.com/SU-SWS/stanford_layouts/pull/4

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/install/core.entity_form_display.node.stanford_news.default.yml [hunk 1]
```diff
@@ -5,13 +5,16 @@ dependencies:
     - field.field.node.stanford_news.body
     - field.field.node.stanford_news.field_s_news_category
     - field.field.node.stanford_news.field_s_news_date
-    - field.field.node.stanford_news.field_s_news_hero_banner
+    - field.field.node.stanford_news.field_s_news_feat_image
```
Comment (sherakama) [inline config/install/core.entity_form_display.node.stanford_news.default.yml]: @pookmish, This field name change will need some of your fancy upgrade path goodness.

Comment (sherakama) [inline config/install/core.entity_form_display.node.stanford_news.default.yml]: Unless the Earth team has yet to start creating news types.

Comment (pookmish) [inline config/install/core.entity_form_display.node.stanford_news.default.yml]: they didnt have any content on production so I already added the field manually on production to keep them unblocked.

Comment (sherakama) [inline config/install/core.entity_form_display.node.stanford_news.default.yml]: It may be non-traditional, but that sounds like an upgrade path to me 😄

**Comment by sherakama** [comment]
@pookmish,

I swapped out the title region for a block in the layout. Please have a review to make sure I didn't mess with yo cheese.

**Comment by pookmish** [comment]
and thats using the 8.x-1.x branch of stanford_layouts. not the branch i made with that separate region right?

**Pull request merged**